src/main/java/model/FruitTransaction.java

```diff
@@ -0,0 +1,58 @@
+package model;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public void setOperation(Operation operation) {
+        this.operation = operation;
+    }
+
+    public String getFruit() {
+        return fruit;
+    }
+
+    public void setFruit(String fruit) {
+        this.fruit = fruit;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public void setQuantity(int quantity) {
+        this.quantity = quantity;
+    }
+
+    public enum Operation {
+        BALANCE("b"),
+        SUPPLY("s"),
+        PURCHASE("p"),
+        RETURN("r");
+
+        private String code;
+
+        Operation(String code) {
+            this.code = code;
+        }
+
+        public String getCode() {
+            return code;
+        }
+
+        public static Operation getFruitByCode(String fruitOperation) {
+
+            for (Operation operation : values()) {
+                if (operation.getCode().equals(fruitOperation)) {
+                    return operation;
+                }
+            }
+            throw new RuntimeException("Can't find code");
```

Comment:
Change the error message in the RuntimeException to provide more information about the error. For example: 'Can't find operation with code: ' + fruitOperation